From a0637f2652aa8f634814366bcb58b1d1e527807d Mon Sep 17 00:00:00 2001
From: Hal Rosenstock <hal@dev.mellanox.co.il>
Date: Thu, 13 Apr 2017 10:48:25 -0400
Subject: [PATCH 2/3] srp_daemon/srp_daemon.c: Eliminate some unneeded code in
 get_shared_pkeys

PathRecords are obtained using SubAdmGet and not SubAdmGetTable
so RMPP is not involved. Eliminate unneeded initialization of MAD
RMPP fields.

Signed-off-by: Hal Rosenstock <hal@mellanox.com>
---
 srp_daemon/srp_daemon.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/srp_daemon/srp_daemon.c b/srp_daemon/srp_daemon.c
index 76fec28..1cf2815 100644
--- a/srp_daemon/srp_daemon.c
+++ b/srp_daemon/srp_daemon.c
@@ -1134,8 +1134,6 @@ static int get_shared_pkeys(struct resources *res,
 
 		/* Mark components: DLID, SLID, PKEY */
 		out_sa_mad->comp_mask = htobe64(1 << 4 | 1 << 5 | 1 << 13);
-		out_sa_mad->rmpp_version = 1;
-		out_sa_mad->rmpp_type = 1;
 		path_rec = (ib_path_rec_t *)out_sa_mad->data;
 		path_rec->slid = htobe16(local_port_lid);
 		path_rec->dlid = htobe16(dest_port_lid);
-- 
1.8.3.1

